engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql

```diff
@@ -132,3 +132,5 @@ INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_vers
 
 -- Add support for windows2019srvNext_64Guest from VMware 7.0.1.0
 INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0.1.0', 'windows2019srvNext_64Guest', 276, now(), 0);
+
+UPDATE `cloud`.`configuration` SET `value` = 20, `default_value` = 20 WHERE `name` = 'default.ui.page.size';
```

Comment:
@davidjumani won't we have to create a 41510to41520 upgrade for it to work for 4.15.1 to 4.15.2 upgrade?

Reply:
If config key exists we shouldn't need it

Reply:
Yes, will make the change